weave/trace/serialization/serialize.py

```diff
@@ -93,8 +93,9 @@ def to_json(
                 for k, v in as_dict.items()
             }
         return fallback_encode(obj)
+    if "val" in encoded:
```

Comment:
would add a comment here noting that it's inline

Reply:
or even a func `is_inline`

Reply:
Adding func